clarity.d.ts

```diff
@@ -35,6 +35,9 @@ interface IConfig {
   // Setting to enable debug features (e.g. console.log statements)
   debug?: boolean;
 
+  // Setting to enable consistency verifications between real DOM and shadow DOM
+  ensureConsistency?: boolean;
```

Comment:
Sounds like a better name, yeah - made the change


---
In reply to: [132250973](https://github.com/Microsoft/clarity-js/pull/25#discussion_r132250973) [](ancestors = 132250973)